### PR TITLE
Setting up the footer and styling in desktop mode

### PR DESCRIPTION
**Pull request by Caroline-CD**

### CODE REVIEW

**Review by Morel-Victor — DISMISSED** [review]
All is good for me !

File: src/components/Footer/Style.jsx [hunk 1]
```diff
@@ -0,0 +1,248 @@
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos/FacebookSquare';
+import { Instagram } from '@styled-icons/boxicons-logos/Instagram';
+import { Twitter } from '@styled-icons/entypo-social/Twitter';
+import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
+
+export const SFooter = styled.footer`
```
Comment (Hacene010) [inline src/components/Footer/Style.jsx]: reconsider your exports (Ask Victor)

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some evils to correct

**Review by Morel-Victor — APPROVED** [review]
Ok for me !!

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**